src/contour/display/ShaderConfig.cpp

```diff
@@ -31,6 +31,8 @@ using namespace std::string_literals;
 namespace contour::display
 {
 
+auto const ShaderLog = logstore::Category("shader", "Logs shader configuration");
```

Comment:
```suggestion
auto const ShaderLog = logstore::Category("gui.shader", "Logs shader configuration");
```

All the GUI code has (IIRC) all logging tags prefixed with `gui.`